SimG4CMS/Calo/src/CaloSD.cc

```diff
@@ -169,6 +170,23 @@ CaloSD::CaloSD(const std::string& name,
 
 CaloSD::~CaloSD() {}
```

Comment:
should it be "default"?

Reply:
This change causes some compilation issues. I leave this for the time being.